third_party/llvm/include/llvm/Demangle/StringView.h

```diff
@@ -0,0 +1,122 @@
+//===--- StringView.h ----------------*- mode:c++;eval:(read-only-mode) -*-===//
```

Comment:
It's certainly confusing that this file references the readme, since the readme references the organization of the LLVM codebase in a way that isn't reflected in our own code...

Reply:
I think it's best to not modify upstream files to avoid adding even more confusion :)

Reply:
Sorry, I definitely agree with that--I wasn't saying that we should change the file.  I was just commenting that it's a little confusing that there's no good way to make the  dependency comments unambiguous.

Upon reflection, I don't think there's anything we can do about that though.  🙃